templates/docs/patterns/lists/index.md

```diff
@@ -7,11 +7,40 @@ context:
 If you want to display lists in a way that is more visually distinctive than the
 standard `<ol>` and `<ul>`, we have 7 list styles at your disposal.
 
+## Horizontal divider
+
+Use the class `.p-list--divided` to add horizontal lines between the items.
+This is meant to be the new default list style please use this going forward.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/lists/lists-dividers/" class="js-example">
+View example of the divided list pattern
+</a></div>
+
+## Dotted with horizontal divider
+
+You can combine both `.is-dotted` and `.p-list--divided` to style a
+list with horizontal dividers and dots.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/lists/lists-dividers-dotted/" class="js-example">
+View example of the dotted divided list pattern
+</a></div>
+
```

Comment:
Would be good to have an example and docs for ordered list with lines as well, now that we ask people to use this style for everything.

So please add another example with numbered list and some short docs section for it, thanks!